include/oneapi/dpl/pstl/utils.h

```diff
@@ -30,7 +30,7 @@
 #    include "hetero/dpcpp/sycl_iterator.h"
 #endif
 
-#if _ONEDPL___cplusplus >= 202002L && __has_include(<bit>)
+#if __cpp_lib_bit_cast >= 201806L && __has_include(<bit>)
```

Comment:
There are two issues with this. 

First, the feature macro is defined, according to the standard, in `<version>` and in `<bit>`. At this point, neither is guaranteed to be included, and each is only available since C++20. That's why the code currently checks for the standard version.

Second, `__cpp_lib_bit_cast` is not the only feature from `<bit>` we need here; see the line 610.